### ISSUE TITLE
ArrayIndexOutOfBoundException in ClassCacheEntry 825

### ISSUE THREAD

**Issue by Agrawalmahin** [aerospike/java-object-mapper#102]
When trying to use the insert function of AeroMapper, we have seen` ArrayIndexOutOfBoundException 0 .`

this is the line in the code , where the error is coming, could it be due to concurrency. 
For the same objects, sometime the request would give a 200 OK response, but sometimes it would fail.
I am using com.aerospike:java-object-mapper:2.0.1 version of this library

`bins[index++] = new Bin(name, aerospikeValue);`

```
ClassCacheEntry.java in getBins at line 825
AeroMapper.java in save at line 237
AeroMapper.java in save at line 202
AeroMapper.java in save at line 196
```

**Comment by roimenashe** [comment]
@Agrawalmahin 
This issue was already opened in #98 and fixed in version 2.0.3 of the Aerospike Java Object Mapper (https://github.com/aerospike/java-object-mapper/releases/tag/2.0.3).

I saw that you already closed this issue, you probably noticed it was already fixed, I just wanted to clarify it for other users that may encounter this issue as well.

Thanks for noticing.

**Comment by Agrawalmahin** [comment]
Yeah, realized it after opening this issue,
Thanks for the help!!
